Prevent use of constant challenge value (#86)
oauthstuff/draft-ietf-oauth-security-topics: recommendations.md

In any case, the PKCE challenge or OpenID Connect `nonce` MUST be
transaction-specific and securely bound to the client and the user agent in
which the transaction was started.

> Authorization servers SHOULD detect and prevent the use of constant PKCE challenge or OpenID Connect `nonce` values.

Reviewer: ```suggestion
Authorization servers are encouraged to make a reasonable effort at detecting and
preventing the use of constant PKCE challenge or OpenID Connect nonce values.
```

In any case, the PKCE challenge or OpenID Connect `nonce` MUST be
transaction-specific and securely bound to the client and the user agent in
which the transaction was started.

> Authorization servers MUST detect and prevent the use of constant
> PKCE challenge or OpenID Connect `nonce` values.

Author: ```suggestion
Authorization servers SHOULD detect and prevent the use of constant PKCE challenge or OpenID Connect `nonce` values.
```

> preventing the use of constant PKCE challenge or OpenID Connect nonce values.

Author: ```suggestion
preventing the use of constant PKCE challenge or OpenID Connect `nonce` values.
```
